Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.10-alpine
+
+RUN apk add --no-cache \
+    redis \
+    git
+
+RUN go get -v \
+    github.com/stretchr/testify \
+    gopkg.in/redis.v5
+
+WORKDIR /go/src/section.io/proxymessage
+COPY *.go ./
+
+RUN gofmt -e -s -d . 2>&1 | tee /gofmt.out && test ! -s /gofmt.out
```

Comment:
can you add `go vet` and `errcheck` too?

Reply:
Updated